ParentPrefixSelectorGuide.md

```diff
@@ -0,0 +1,51 @@
+# A guide of `ParentPrefixSelector` in `PrefixClaim`
+
+There are 2 ways to make a Prefix claim:
+- provide a `parentPrefix`
+- provide a `parentPrefixSelector`
+
+In this documentation, we will focus on the `parentPrefixSelector` only.
```

Comment:
> Please add a note on how it behaves in case both are set. Which one takes precedence?

In the current design, CRD validation will reject on applying.